.ruby-version

```diff
@@ -1 +1 @@
-ruby-3.2.3
+ruby-3.2.2
```

Comment:
Please make the relevant changes in `.tool_versions` as well.

Reply:
done